### ISSUE TITLE
Extremely low contrast between cursorline and comment color

### ISSUE THREAD

**Issue by drzel** [arcticicestudio/nord-vim#48]
Really love this colorscheme, far and beyond any other. I use it wherever I can.

I realise it's subjective, but it's pretty hard to read comments as it is - and very tough under the cursorline.

**Comment by b0o** [comment]
I agree. If there is any tiny amount of glare/excess light on my screen, comments are basically unreadable.

**Comment by arcticicestudio** [comment]
The low contrast is by design of Nord and has been selected specifically to not disturb code flow and keep the focus of the developer on the code.

It is totally fine that you not agree to all highlighting color assignments, everyone has different preferences. That is the reason why themes exist :smile: 
You can customize the theme via  your `~/.vimrc` by pasting the following snippet somewhere before you set the color scheme (`colorscheme nord`) and than add your custom syntax highlighting rules you want to override:

```vim
augroup nord-overrides
  autocmd!
  autocmd ColorScheme nord highlight Comment ctermfg=<COLOR> guifg=<COLOR>
augroup END
```

The overriden rules will only be applied for Nord and won't affect other themes.

This is a reference table if you'd like to use the same increased contrast values as provided by the [Nord Atom Syntax accessibility custom comment contrast theme setting feature](https://github.com/arcticicestudio/nord-atom-syntax/pull/47) which are calculated using the LESSCSS [`lighten`](http://lesscss.org/functions/#color-operations-lighten) function.

| Increased by | Calculated value |
| --- | --- |
| 1% | `#4e586d` |
| 2% | `#505b70` |
| 3% | `#525d73` |
| 4% | `#556076` |
| 5% | `#576279` |
| 6% | `#59647c` |
| 7% | `#5b677f` |
| 8% | `#5d6982` |
| 9% | `#5f6c85` |
| 10% | `#616e88` |
| 11% | `#63718b` |
| 12% | `#66738e` |
| 13% | `#687591` |
| 14% | `#6a7894` |
| 15% | `#6d7a96` |
| 16% | `#6f7d98` |
| 17% | `#72809a` |
| 18% | `#75829c` |
| 19% | `#78859e` |
| 20% | `#7b88a1` |

**Comment by drzel** [comment]
Thanks so much for taking the time to explain this. I've been looking into how to do exactly what you describe.

**Comment by b0o** [comment]
Thank you! I find that bumping the comment color up just 5% to `#576279` makes a world of difference.

**Comment by arcticicestudio** [comment]
@b0o I've merged #56 so you can now change the percent with the new configuration. If you' re using [vim-plug](https://github.com/junegunn/vim-plug) you can easily change the plugin to use the `develop` branch so you can use this feature before version [0.7.0]() is released.

Add

```vim
Plug "arcticicestudio/nord-vim", { 'branch': "develop" }
```

to your `~/.vimrc` and run `:PlugInstall`.
